src/test/java/com/powsybl/openloadflow/Ieee14Test.java

```diff
@@ -98,18 +98,18 @@ void testDc() {
         result = loadFlowRunner.run(network, parameters);
         assertEquals(LoadFlowResult.ComponentResult.Status.CONVERGED, result.getComponentResults().get(0).getStatus());
         assertAngleEquals(0, network.getBusView().getBus("VL1_0"));
-        assertAngleEquals(-5.0125806, network.getBusView().getBus("VL2_0"));
-        assertAngleEquals(-12.955848, network.getBusView().getBus("VL3_0"));
-        assertAngleEquals(-10.587248, network.getBusView().getBus("VL4_0"));
-        assertAngleEquals(-9.091747, network.getBusView().getBus("VL5_0"));
-        assertAngleEquals(-15.229412, network.getBusView().getBus("VL6_0"));
-        assertAngleEquals(-13.957433, network.getBusView().getBus("VL7_0"));
-        assertAngleEquals(-13.957433, network.getBusView().getBus("VL8_0"));
-        assertAngleEquals(-16.025846, network.getBusView().getBus("VL9_0"));
-        assertAngleEquals(-16.313486, network.getBusView().getBus("VL10_0"));
-        assertAngleEquals(-15.976867, network.getBusView().getBus("VL11_0"));
-        assertAngleEquals(-16.340761, network.getBusView().getBus("VL12_0"));
-        assertAngleEquals(-16.510537, network.getBusView().getBus("VL13_0"));
-        assertAngleEquals(-17.536792, network.getBusView().getBus("VL14_0"));
+        assertAngleEquals(-5.013434, network.getBusView().getBus("VL2_0"));
```

Comment:
@zamarrenolm @marqueslanauja Is it expected to not have same result for DC calculation with no transformer ratio?

Reply:
if `setDcUseTransformerRatio` is set to `false` the result for DC calculation will not be the same as the ratio must be considered to have the exact per unit conversion.

Reply:
In the IEEE 14 example we have 2 lines with different Vnom:
---------- Line L7-8-1 has Vnom1 = 14.0 and Vnom2 = 20.0------------------------
---------- Line L7-9-1 has Vnom1 = 14.0 and Vnom2 = 12.0------------------------

Reply:
The two ways of per uniting are strictly equivalent for AC use.

If the option with no transformer ratio is active, I suppose that in the previous way of per uniting setR1(1 / Transformers.getRatioPerUnitBase(line)) is transformed into 1.
In the new way of per-uniting, the option has no effect since R1 is always one by construction.

Reply:
the new way of per uniting seems to keep more consistance between AC model and DC model approx